octodns_powerdns/__init__.py

```diff
@@ -364,7 +384,7 @@ def populate(self, zone, target=False, lenient=False):
             exists = True
             for rrset in resp.json()['rrsets']:
                 _type = rrset['type']
-                if _type == 'SOA':
+                if _type not in self.SUPPORTS:
```

Comment:
Ah, `LUA` needs to be checked with the provider specific prefix:

```suggestion
                _provider_specific_type = f'PowerDnsProvider/{_type}'
                if (
                    _type not in self.SUPPORTS
                    and _provider_specific_type not in self.SUPPORTS
                ):
```